src/main/java/org/aitesting/passenger/model/PassengerDto.java

```diff
@@ -0,0 +1,35 @@
+package org.aitesting.passenger.model;
```

Comment:
This DTO is not needed, this is exactly what the Passenger model is.

Reply:
Wouldn't the DTO be needed for POST request and the Passenger model for saving to the database?